docs/source/json-api/index.rst

```diff
@@ -626,3 +612,261 @@ Contract Found Response
             "templateId": "11c8f3ace75868d28136adc5cfc1de265a9ee5ad73fe8f2db97510e3631096a2:Account:Account"
         }
     }
+
+
+Contract Search, All Templates
+==============================
+
+List all currently active contracts for all known templates. Note that the retrieved contracts do not get persisted into query store database.
```

Comment:
Is there any explanation of what the "query store database" is? I couldn't find any.